atd-vze/src/views/Crashes/Recommendations/RecommendationMultipleSelectDropdown.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Multiselect from "multiselect-react-dropdown";
+import "../crash.scss";
+
+const RecommendationMultipleSelectDropdown = ({
+  options,
+  onOptionClick,
+  onOptionRemove,
+  partners,
+  fieldConfig,
+  field,
+}) => {
+  // Trigger mutation call to add partner upon click
+  const handleOptionClick = selectedItem => {
+    console.log(selectedItem);
```

Comment:
Would it be alright to remove this `console.log()` and any others that are tagging along?

Reply:
oops thanks for catching that! i literally just added that then forgot to remove it before i pushed 🤦